### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by 0exp**
- rubocop: 0.60.0
- rubocop-rspec: 1.30.1
- gem version: 0.60.0

### CODE REVIEW

File: rubocop/rubocop.gemspec [hunk 1]
```diff
@@ -6,6 +6,10 @@ Gem::Specification.new do |spec|
   # x.y.z.t
   # Where x.y.z = rubocop version
   # t is incremental
+
+  # TODO: make as a separated gem hosted via rubygems
```
Comment (tycooon) [inline rubocop/rubocop.gemspec]: What do you mean?

File: rubocop/rubocop.gemspec [hunk 2]
```diff
@@ -6,6 +6,10 @@ Gem::Specification.new do |spec|
...
+  # TODO: rubocop-gem-non-related versioning
```
Comment (tycooon) [inline rubocop/rubocop.gemspec]: Why?

Comment (TrumpClone) [reply rubocop/rubocop.gemspec]: Rubocop ruleses is our "version" - not a gem.
Change cop options => Change version - How? (problem) (and this is the main point of this gem i think)
Change gem dependency => Change version - OK.

File: rubocop/rubocop.gemspec (hunk 1 above)
Comment (TrumpClone) [reply rubocop/rubocop.gemspec]: Push to the rubygems and install via it so we can simply use any versions of this gem and giving a full control over it.

Comment (TrumpClone) [reply rubocop/rubocop.gemspec]: Full controll over the gem and CI with better versioning described below

Comment (TrumpClone) [reply rubocop/rubocop.gemspec]: at this moment we have only github-hosted project with "commit-based" versioning :\

File: rubocop/rubocop.gemspec (hunk 2 above)
Comment (JelF) [reply rubocop/rubocop.gemspec]: > Change cop options => Change version - How? (problem) (and this is the main point of this gem i think)

It is described in comment above: you should increment "t"

RTFM

File: rubocop/rubocop.gemspec (hunk 1 above)
Comment (JelF) [reply rubocop/rubocop.gemspec]: > at this moment we have only github-hosted project with "commit-based" versioning :\

There is rubygems-hosted gem: https://rubygems.org/gems/rubocop-config-umbrellio.
`umbrellio` (@tycooon know who owns that account) has an access to push it

Comment (TrumpClone) [reply rubocop/rubocop.gemspec]: i know. but ok :)

File: rubocop/rubocop.gemspec (hunk 2 above)
Comment (TrumpClone) [reply rubocop/rubocop.gemspec]: Weeeeeeell, Ok

**Review by tycooon — APPROVED** [review]

**Pull request merged**